packages/live-share-inking/LICENSE

```diff
@@ -0,0 +1,87 @@
+MICROSOFT SOFTWARE LICENSE TERMS 
```

Comment:
qq: Do the other samples need this license, or is this a special case due to inking?

Reply:
I borrowed this header from the main SDK and other samples.

Reply:
my bad; the one sample I checked didn't have a license haha. I can do a PR to add it some other time